build.gradle

```diff
@@ -1350,7 +1352,8 @@ project(':connect:runtime') {
     testCompile libs.powermockJunit4
     testCompile libs.powermockEasymock
 
-    testCompile project(":connect:json")
+    testCompile project(':connect:json')
```

Comment:
This shouldn't be necessary anymore since we now have this as a `compile` dependency.